src/Catty/Defines/UIDefines.h

```diff
@@ -93,12 +93,11 @@ typedef NS_ENUM(NSUInteger, kBrickCategoryType) {
     kPhiroBrick                = 8,
     kPenBrick                  = 9,
     kEmbroideryBrick           = 10,
-    kInvisible                = 99,
-    kFavouriteBricks           = 0
+    kInvisible                 = 99,
+    kRecentlyUsedBricks        = 0
 };
 
-#define kMinFavouriteBrickSize 5
-#define kMaxFavouriteBrickSize 10
+#define kRecentlyUsedSize 10
```

Comment:
Please also add a constant for the minimum size which needs to be available (set to 1 right now but this should be flexible).